tests/index.test.ts

```diff
@@ -0,0 +1,89 @@
+import { ExtensibilitySdk, Message, OAuthDialogCompletedMessage } from '../src/index';
+import runtime from '../src/sdk/RuntimeContext';
+import oauthService from '../src/sdk/services/oauthService';
+
+describe('sdk tests', () => {
+  let sdk: ExtensibilitySdk;
+
+  describe.only('receiving messages', () => {
```

Comment:
It should be probably w/o `only`
```suggestion
  describe('receiving messages', () => {
```

Reply:
removed